src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -187,16 +191,36 @@ namespace AppInstaller::CLI::Workflow
                 }
                 else
                 {
-                    entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(entryPath, targetPath, {})));
+                    if (isSymlinkCreationSupported)
+                    {
+                        entries.emplace_back(std::move(PortableFileEntry::CreateFileEntry(entryPath, targetPath, {})));
+                    }
+                    else
+                    {
+                        // Find matching nested installer file and rename the symlink target with the corresponding command alias if available.
+                        for (const auto& nestedInstallerFile : nestedInstallerFiles)
+                        {
+                            if (relativePath == std::filesystem::path{ ConvertToUTF16(nestedInstallerFile.RelativeFilePath) })
+                            {
+                                if (!nestedInstallerFile.PortableCommandAlias.empty())
+                                {
+                                    std::filesystem::path targetWithCommandAlias = targetInstallDirectory / ConvertToUTF16(nestedInstallerFile.PortableCommandAlias);
```

Comment:
Probably we'll want to keep the nested path structure and only replace the filename to be safe